homeassistant/components/light/lw12wifi.py

```diff
@@ -0,0 +1,166 @@
+"""
+Support for Lagute LW-12 WiFi LED Controller.
+
+Configuration example:
```

Comment:
Move the example to the docs PR. Add a link here to the docs page as done in other platforms.

Reply:
@fabaff was so friendly and already created a PR to add the example configuration [#5264](https://github.com/home-assistant/home-assistant.github.io/pull/5264).

Reply:
Good. Replace these instructions here with the link to the future page in the docs. See other platforms for examples.